<?xml version="1.0"?>
<launch>
  <node name="fake_publisher" pkg="tcp_ros" type="fake_publisher" output="screen"/>

</launch>
